app/scripts/modules/azure/serverGroup/configure/wizard/basicSettings/basicSettings.html

```diff
@@ -51,8 +51,8 @@
         <div class="col-md-9">
           <ui-select class="form-control input-sm" required ng-model="command.imageName" on-select="basicSettingsCtrl.imageChanged($item)">
             <ui-select-match placeholder="Pick an image">{{$select.selected.imageName}}</ui-select-match>
-            <ui-select-choices repeat="image.imageName as image in command.images | anyFieldFilter: {imageName: $select.search, ami: $select.search } | orderBy: '-imageName'">
-              <span ng-bind-html="image.imageName | highlight: $select.search"></span>
+            <ui-select-choices repeat="image.imageName as image in command.images | regional:command.region ">
```

Comment:
Does this still filter the images based on the user typing into the search field?